<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PydaqGlobal</class>
 <widget class="QMainWindow" name="PydaqGlobal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>747</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PYDAQ Global</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources_1.qrc">
    <normaloff>:/imgs/imgs/favicon.ico</normaloff>:/imgs/imgs/favicon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(64, 64, 64);
}

QTabWidget::pane { 
   border: 1px solid rgb(166, 166, 166);
}

QTabBar::tab {
  	background-color: rgb(77, 77, 77);
 }

 QTabBar::tab:selected {
  	background-color: rgb(140, 140, 140);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:selected:hover {
  	background-color: rgb(140, 140, 140);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:hover {
  	background-color: rgb(109, 109, 109);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:middle {
	border-right: 1px dashed rgb(166, 166, 166);
	border-left: 1px dashed rgb(166, 166, 166);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:last {
	border-top-right-radius: 10px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:first {
	border-top-left-radius: 10px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }


QComboBox QAbstractItemView {
    background-color: rgb(77, 77, 77);
}

QComboBox QAbstractItemView::item:focus{
    background-color: rgb(140, 140, 140);
}

QDoubleSpinBox{
	background-color: rgb(77, 77, 77);
	
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}

QDoubleSpinBox::up-button{
    image: url(:/imgs/imgs/drop_up_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::up-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::up-button:hover{
	background-color: rgb(0, 50, 0);
}

QDoubleSpinBox::down-button{
    image: url(:/imgs/imgs/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::down-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::down-button:hover{
	background-color: rgb(0, 50, 0);
}

QWidget#centralwidget{
	background-color: rgb(64, 64, 64);
}

QWidget{
	color: rgb(255, 255, 255);
}

QComboBox{
	background-color: rgb(77, 77, 77);
	
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}


QComboBox::drop-down{
	image: url(:/imgs/imgs/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 2px solid rgb(127, 167, 127);
	border-left: 2px solid rgb(127, 167, 127);

	border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QComboBox::drop-down:hover{
	background-color: rgb(0, 50, 0);
}

QComboBox::drop-down:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QPushButton{
	background-color: rgb(0, 79, 0);

	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);

	
	font: 12pt &quot;Helvetica&quot;;
	text-align:center;
}

QWidget{
	font: 12pt &quot;Helvetica&quot;;
}

QPushButton:hover{
	background-color: rgb(0, 50, 0);
}

QPushButton:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QLineEdit{
	background-color: rgb(77, 77, 77);
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}

QRadioButton::indicator{
	border-radius: 6px;
	border-top: 1.5px solid rgb(0, 0, 0);
	border-left: 1.5px solid rgb(0, 0, 0);

	border-bottom: 1.5px solid rgb(160, 160, 160);
	border-right: 1.5px solid rgb(160, 160, 160);
}

QRadioButton::indicator::checked{
	background-color: white;
}

QRadioButton::indicator::unchecked:hover{
	background-color: #9F9F9F;
}

QRadioButton::indicator::pressed{
	border: 1.5px solid #505050
}

QPushButton#reload_devices{
	image: url(:/imgs/imgs/reload.png);
	width: 11px;
	background-color: rgb(0, 79, 0);

	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);

	
	font: 12pt &quot;Helvetica&quot;;
	text-align:center;
}

QPushButton#reload_devices:hover{
	background-color: rgb(0, 50, 0);
}

QPushButton#reload_devices:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QPushButton#logo{
	background-color: rgb(64, 64, 64);
	border: 0px;
}

QPushButton#logo:hover{
	background-color: rgb(166, 166, 166);
	border: 1px solid rgb(190, 190, 190) ;
}

QPushButton#logo:pressed{
	background-color: rgb(166, 166, 166);
	border: 1px solid rgb(255, 255, 255) ;
}

QScrollBar:vertical {
     border: 1px solid rgb(140, 140, 140);
     background: rgb(140, 140, 140);
     width: 17px;
     margin: 17px 0 17px 0;
 }
QScrollBar::handle:vertical {
     background: rgb(0, 79, 0);
     min-height: 20px;
 }
QScrollBar::add-line:vertical {
	image: url(:/imgs/imgs/drop_down_arrow.png);
     border: 1px solid rgb(140, 140, 140);
     background: rgb(0, 79, 0);
     height: 15px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

QScrollBar::sub-line:vertical {
	image: url(:/imgs/imgs/drop_up_arrow.png);
     border: 1px solid rgb(140, 140, 140);
     background: rgb(0, 79, 0);
     height: 15px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar::add-line:vertical:pressed {
    border: 1px solid rgb(255, 255, 255);
	background: rgb(0, 79, 0)
 }

QScrollBar::sub-line:vertical:pressed {
	border: 1px solid rgb(255, 255, 255);
	background: rgb(0, 79, 0)
 }

QScrollBar::add-line:vertical:hover {
     background-color: rgb(0, 50, 0);
 }

QScrollBar::sub-line:vertical:hover {
    background-color: rgb(0, 50, 0);
 }

QScrollBar::handle:vertical:pressed {
    border: 1px solid rgb(255, 255, 255);
	background: rgb(0, 79, 0)
 }

QScrollBar::handle:vertical:hover {
    background-color: rgb(0, 50, 0);
 }

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

QScrollArea{
	border: none;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="logo">
      <property name="minimumSize">
       <size>
        <width>186</width>
        <height>147</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>186</width>
        <height>147</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Go to PYDAQ Documentation</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources_1.qrc">
        <normaloff>:/imgs/imgs/logo.png</normaloff>:/imgs/imgs/logo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>182</width>
        <height>143</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Arduino</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">interface_radio</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>NIDAQ</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">interface_radio</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QTabWidget" name="nidaq_tabs">
         <property name="tabPosition">
          <enum>QTabWidget::TabPosition::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>false</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="get_data_nidaq">
          <attribute name="title">
           <string>Get Data</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="GetData_NIDAQ_Widget" name="get_nidaq_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="send_data_nidaq_tab">
          <attribute name="title">
           <string>Send Data</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_5">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <widget class="SendData_NIDAQ_Widget" name="send_nidaq_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="step_response_nidaq_tab">
          <attribute name="title">
           <string>Step Response</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="StepResponse_NIDAQ_Widget" name="step_nidaq_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="get_model_nidaq_tab">
          <attribute name="title">
           <string>Get Model</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_7">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_15">
               <item row="0" column="0">
                <widget class="GetModel_Nidaq_Widget" name="getm_nidaq_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pid_control_nidaq_tab">
          <attribute name="title">
           <string>PID Control</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_20">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_10">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_9">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_21">
               <item row="0" column="0">
                <widget class="PID_Control_NIDAQ_Widget" name="widget_4" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTabWidget" name="arduino_tabs">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::TabShape::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::TextElideMode::ElideNone</enum>
         </property>
         <widget class="QWidget" name="get_data_arduino_tab">
          <attribute name="title">
           <string>Get Data</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_4">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_6">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <widget class="GetData_Arduino_Widget" name="get_ino_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="send_data_arduino_tab">
          <attribute name="title">
           <string>Send Data</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_5">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_7">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <widget class="SendData_Arduino_Widget" name="send_ino_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="step_response_arduino_tab">
          <attribute name="title">
           <string>Step Response</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_6">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_11">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_14">
               <item row="0" column="0">
                <widget class="StepResponse_Arduino_Widget" name="step_ino_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="get_model_arduino_tab">
          <attribute name="title">
           <string>Get Model</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_8">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_17">
               <item row="0" column="0">
                <widget class="GetModel_Arduino_Widget" name="getm_arduino_placeholder" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pid_control_arduino_tab">
          <attribute name="title">
           <string>PID Control</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_9">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_8">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>418</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_19">
               <item row="0" column="1">
                <widget class="PID_Control_Arduino_Widget" name="widget_3" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuBenchmarking_2">
     <property name="title">
      <string>Benchmarking</string>
     </property>
     <addaction name="actionArduino_2"/>
     <addaction name="actionNIDAQ_2"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuBenchmarking_2"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionArduino">
   <property name="text">
    <string>Arduino</string>
   </property>
  </action>
  <action name="actionNIDAQ">
   <property name="text">
    <string>NIDAQ</string>
   </property>
  </action>
  <action name="actionAbout_PYDAQ">
   <property name="text">
    <string>About PYDAQ</string>
   </property>
  </action>
  <action name="actionArduino_2">
   <property name="text">
    <string>Arduino</string>
   </property>
  </action>
  <action name="actionNIDAQ_2">
   <property name="text">
    <string>NIDAQ</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GetData_Arduino_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.getdata_arduino_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SendData_Arduino_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.senddata_arduino_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StepResponse_Arduino_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.stepresponse_arduino_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StepResponse_NIDAQ_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.stepresponse_nidaq_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SendData_NIDAQ_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.senddata_nidaq_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GetData_NIDAQ_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.getdata_nidaq_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GetModel_Arduino_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.getmodel_arduino_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GetModel_Nidaq_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.getmodel_nidaq_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PID_Control_Arduino_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.pid_control_arduino_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PID_Control_NIDAQ_Widget</class>
   <extends>QWidget</extends>
   <header>..guis.pid_control_nidaq_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources_1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>arduino_tabs</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>nidaq_tabs</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>arduino_tabs</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>nidaq_tabs</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="interface_radio"/>
 </buttongroups>
</ui>
